vhdbuilder/packer/init-variables.sh

```diff
@@ -211,6 +211,15 @@ if [ -n "${WINDOWS_MSI_RESOURCE_STRING}" ]; then
 	windows_msi_resource_strings+=(${WINDOWS_MSI_RESOURCE_STRING})
 fi
 
+linux_msi_resource_ids=()
```

Comment:
does this need to be an array? I think it would only ever hold one value for linux

Reply:
It created an issue for windows when it was not an array when MSI was not specified, i haven't come across the same for Linux but no harm in keeping it as an array - consistent and expadable?